### PR TITLE
[ETL-275] Add recover production account under ScienceProdOU

### PR DESCRIPTION
**Pull request by rxu17**
Hello,
I'm opening this PR to create a production AWS account. This production account paired with the development account will house all the ETL work required for the RECOVER project, such that all the work done in these accounts will be charged to the RECOVER grant.

See [PR #649](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/649) for the PR creating the corresponding development account.

PR Checklist:
[X] Describe and explain your intentions for this change
[X] Setup pre-commit and run the validators (info in README.md)

### CODE REVIEW

**Comment by rxu17** [comment]
@thomasyu888

**Comment by thomasyu888** [comment]
Nit: @rxu17 Can you make a tiny change in the name of this PR from `ScienceDevOU` to `ScienceProdOU`

File: org-formation/organization.yaml [hunk 1]
```diff
@@ -563,6 +564,22 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         <<: !Include ./_platform_org_tags.yaml
 
+  RecoverProdAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-recover-prod
+      AccountId: '634761300905'
```
Comment (brucehoff) [inline org-formation/organization.yaml:571]: Omit this line (`AccountId: '634761300905'`)?

File: org-formation/organization.yaml [hunk 2]
```diff
@@ -563,6 +564,22 @@ Organization:
...
+      RootEmail: aws-recover-prod@sagebase.org
```
Comment (brucehoff) [inline org-formation/organization.yaml:572]: I don't see this alias in Google Groups.  Please see the note in step 2 here: https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2058878986/Bootstrapping+AWS+Project+Accounts

**Review by brucehoff — CHANGES_REQUESTED** [review]

File: org-formation/organization.yaml (hunk 2 above)
Comment (thomasyu888) [reply org-formation/organization.yaml:572]: Done

File: org-formation/organization.yaml (hunk 1 above)
Comment (rxu17) [reply org-formation/organization.yaml]: Done

File: org-formation/organization.yaml (hunk 2 above)
Comment (brucehoff) [reply org-formation/organization.yaml:571]: "Who can post" must be "Anyone on web", lest AWS cannot send email to the endpoint: https://groups.google.com/a/sagebase.org/g/aws-recover-prod/settings?hl=en

**Comment by brucehoff** [comment]
LGTM but I'd like @zaro0508 to give this a review, since this is my first time reviewing an account creation PR.

File: org-formation/organization.yaml (hunk 2 above)
Comment (thomasyu888) [reply org-formation/organization.yaml:571]: Done!

**Review by zaro0508 — APPROVED** [review]
LGTM.  @brucehoff there are a few additional manual steps to creating an AWS account. for full instructions take a look at https://sagebionetworks.jira.com/wiki/spaces/IT/pages/745373697/Setup+AWS+Accounts

**Review by brucehoff — APPROVED** [review]

**Pull request merged**